src/ActionModal/components/ModalPropose.vue

```diff
@@ -217,7 +226,17 @@ export default {
       amount: {
         required: x => !!x && x !== `0`,
         decimal,
-        between: between(SMALLEST, this.balance.amount)
+        max: x => Number(x) <= this.balance.amount,
+        min: x => Number(x) >= SMALLEST,
```

Comment:
do we need a min if we have `maxDecimals` and `required`?

Reply:
yes because of negative numbers

Reply:
ahhhh